Automattic/pocket-casts-android
# Playback 2023 - Top listened categories story

| 📘 Part of: #1463 | 🎨 Designs: lH66LwxxgG8btQ8NrM0ldx-fi-1599_20385 |
|:---:|:---:|

## Description
Adds the 2023 visual for the Top Categories Story.

## Testing Instructions
1. Make sure you're logged in to an account that has a few episodes listened
3. Go to Profile
4. Tap the EOY image
5. Skip to the fifth story
5. ✅ Check that the story looks good, that it shows your top categories and for the top one, the number of episodes listened to
6. Share this story
8. ✅ The story image asset should look good

## Screenshots or Screencast 

Pixel 7 (Normal display) | Pixel 7 (Large display) 
|----|---|
|<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/9bd5b9f6-220a-425a-bf15-862a9d482f10"/>|<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/61d38758-f166-4e40-92e1-8c6d2dedfbf3c"/>|

Nexus 9 Tablet
|---|
|<img width=400 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/08f40efe-7ae0-4e7f-8e27-2ddd533c4c11"/>

## Checklist
 
#### I have tested any UI changes...
